### ISSUE TITLE
Loadgen crashing?

### ISSUE THREAD

**Issue by watson** [elastic/opbeans-loadgen#4]
Since a few days ago, the load generator have stopped sending requests to any of the opbeans. It looks like the docker container simply shuts down after being started by compose.py.

**Comment by watson** [comment]
Logs:

```
localtesting_6.4.0_opbeans-load-generator | Traceback (most recent call last):
localtesting_6.4.0_opbeans-load-generator |   File "generate_procfile.py", line 26, in <module>
localtesting_6.4.0_opbeans-load-generator |     create_procfile(*sys.argv[1:])
localtesting_6.4.0_opbeans-load-generator |   File "generate_procfile.py", line 6, in create_procfile
localtesting_6.4.0_opbeans-load-generator |     rpms = {service_name: int(rpm) for service_name, rpm in (entry.split(':') for entry in rpm_env_string.split(','))}
localtesting_6.4.0_opbeans-load-generator |   File "generate_procfile.py", line 6, in <dictcomp>
localtesting_6.4.0_opbeans-load-generator |     rpms = {service_name: int(rpm) for service_name, rpm in (entry.split(':') for entry in rpm_env_string.split(','))}
localtesting_6.4.0_opbeans-load-generator | ValueError: not enough values to unpack (expected 2, got 1)
```

**Comment by beniwohli** [comment]
should be fixed by 00d49f28fdafe
